scripts/deploy_ui.sh

```diff
@@ -27,6 +27,7 @@ mkdir -p build
 
 print_log "Starting ui"
 
+${CONTAINER_COMMAND} pull quay.io/ocpmetal/ocp-metal-ui:latest
```

Comment:
why?

Reply:
because the host might have an older image (with the same tag) and it will not pull the updated one